File: android/app/src/main/java/com/TMR24/MotionStudy/StartActivity.java
package com.TMR24.MotionStudy;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class StartActivity extends AppCompatActivity {
    private Button buttonGo;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_start);
    }
    public void buttonGo(View view)
    {
        Intent i = new Intent(StartActivity.this, MainActivity.class);
        startActivity(i);
    }
}

File: android/app/src/main/java/com/TMR24/MotionStudy/StartActivity.kt
package com.TMR24.MotionStudy

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.content.Intent
import android.view.View
import android.widget.Button


class StartActivity : AppCompatActivity() {
    private val buttonGo: Button? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_start)
    }

    fun buttonGo(view: View?) {
        val i = Intent(this@StartActivity, MainActivity::class.java)
        startActivity(i)
    }
}